documentation/asciidoc/topics/con_cli_connections.adoc

```diff
@@ -0,0 +1,19 @@
+= Connecting with the {brandname} CLI
+Use the command line interface (CLI) to connect to your {brandname} cluster and create caches and perform administrative operations.
+
+The CLI is available as part of the server distribution, which you can run on your local host to establish remote connections to {brandname} clusters on {openshiftshort}.
+
+[NOTE]
+====
+It is possible to open a remote shell to a {brandname} node and access the CLI.
+
+[source,options="nowrap",subs=attributes+]
+----
+$ {kubectl_exec}
+----
+
+However, this approach has drawbacks such as:
+
+* The CLI consumes memory allocated to the container, which can lead to out of memory exceptions.
+* Caches that you create this way are not replicated to other nodes.
```

Comment:
Not true. If you create a cache(not local-cache)/template/counter, the server will ensure that it's configuration is replicated to all nodes in the cluster.